index.html

```diff
@@ -1,100 +1,113 @@
+  </head>
 
-<head>
-  <meta charset="UTF-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
-    integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous" />
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.2/font/bootstrap-icons.min.css" />
-  <link rel="stylesheet" href="./assets/css/style.css" />
-  <title>Biblio</title>
-</head>
-
-<body>
-  <header class="container-fluid d-flex" id="header">
-    <h1 style="margin: 20px">Biblio</h1>
-    <div class="row mx-auto align-items-center">
-      <div class="col">
-        <i class="bi bi-search"></i>
-        <label for="search"><input type="text" class="w-100 rounded-pill py-3" id="search" placeholder="Search a word"
-            name="search" /></label>
+  <body>
+    <header class="container-fluid d-flex" id="header">
+      <h1 style="margin: 20px">Biblio</h1>
+      <div class="row mx-auto align-items-center">
+        <div class="col">
+          <i class="bi bi-search"></i>
+          <label for="search"
+            ><input
+              type="text"
+              class="w-100 rounded-pill py-3"
+              id="search"
+              placeholder="Search a word"
+              name="search"
+          /></label>
+        </div>
       </div>
-    </div>
-  </header>
+    </header>
 
-  <main class="container mt-5 d-flex flex-column flex-md-row gx-2 mx-sm-1 mx-md-auto">
-    <div class="container">
-      <section id="result" class="my-1">
-        <h2>Definition</h2>
-        <p id="definition">
-          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dignissimos
-          rerum veritatis doloremque doloribus, earum libero repudiandae tempore
-          quos saepe reiciendis qui impedit repellat voluptatem amet, nostrum
-          soluta rem quis voluptate. Neque necessitatibus quisquam, officiis,
-          sint quod quaerat, doloribus velit at quam rem hic id eligendi cum non.
-        </p>
-      </section>
+    <main class="container mt-5 d-flex flex-column flex-md-row gx-2 mx-sm-1 mx-md-auto">
+      <div class="container">
+        <section id="result" class="my-1">
+          <h2>Definition</h2>
+          <p id="definition">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dignissimos rerum veritatis
+            doloremque doloribus, earum libero repudiandae tempore quos saepe reiciendis qui impedit
+            repellat voluptatem amet, nostrum soluta rem quis voluptate. Neque necessitatibus
+            quisquam, officiis, sint quod quaerat, doloribus velit at quam rem hic id eligendi cum
+            non.
+          </p>
+        </section>
 
-      <section id="pronunciation" class="my-1">
-        <h2>Pronunciation</h2>
-        <audio controls>
-          <source src="horse.mp3" type="audio/mpeg" />
```

Comment:
I removed the "horse.mp3" from the audio source element's src attribute (it was causing a 404 error in the console when trying to fetch it)